apps/web/src/features/calendar/calendar.css

```
@@ -451,6 +502,14 @@
 	color: var(--color-surface);
 }
 
+.calendar-view
+	.fc
+	.calendar-event-content.calendar-event-content-selected
+	.calendar-event-response-dot {
+	border-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)